code/game/data_huds.dm

```diff
@@ -300,7 +300,7 @@
 			if(!HAS_TRAIT(src, TRAIT_PSY_DRAINED))
 				infection_hud.icon_state = "psy_drain"
 			if(HAS_TRAIT(src, TRAIT_UNDEFIBBABLE ))
-				hud_list[HEART_STATUS_HUD].icon_state = "still_heart"
+				hud_list[HEART_STATUS_HUD].icon_state = "heart_dnr"
 				status_hud.icon_state = "huddead"
```

Comment:
?

Reply:
more descriptive name

Reply:
This is a lie you replaced it with another icon
Do not lie to me please